app/assets/stylesheets/custom.scss

```diff
@@ -34,10 +34,26 @@ body {
 .panel-body h1 {
   font-weight: 900;
 }
-.dad-testamonial {
+.dad-testimonial {
   font-style: italic;
 }
 
+.inline {
+  display: inline;
+}
+
+.dad-admin-order-table form {
+  display: inline;
+}
+
+.hidden {
+  display: none;
+}
+
+.dad-admin-filter-color {
+  background-color: #DDDDDD;
```

Comment:
Color literals like `#DDDDDD` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#DDDDDD` should be written as `#dddddd`